roundedcube.scad

```diff
@@ -0,0 +1,149 @@
+use<FunctionalOpenSCAD/functional.scad>
```

Comment:
Can you create a separate PR for independent features?  
When you submit a PR it accounts for ALL differences between my branch and yours.  So typically if you have multiple changes you are working on concurrently, you would create a branch for each feature/task.

Reply:
What's the best way to do that now that I've got both in on my local version?

Reply:
Probably easiest to delete roundedcube.scad in your master, then we can focus on this PR striclty for the triangulation feature.
Since this PR came from your master branch, any updates to your master will update this PR automatically, until the PR is closed.

Then create a new "rounded_cube" branch. Ideally you would create this branch this from your master at from the commit just before you added triangulation.
git branch branchname <sha1-of-commit>
Your roundedcube.scad will still be in your git history, and you can re-add it in this branch.